lib/util/cmcd_manager.js

```diff
@@ -104,33 +141,42 @@ shaka.util.CmcdManager = class {
    * Apply CMCD data to a segment request
    *
    * @param {!shaka.extern.Request} request
-   * @param {shaka.util.CmcdManager.SegmentInfo} segmentInfo
```

Comment:
If I'm not missing something, I think you have replaced every reference to `shaka.util.CmcdManager.SegmentInfo`. You can probably just rid of the type entirely.

Reply:
Removed along with `ManifestInfo`